### DISCUSSION TITLE
SOLVED Translation file

### DISCUSSION THREAD

**Discussion by GuilhermeDeLucca** [LiveHelperChat/livehelperchat#1829] [General]
I'm having trouble translating this value! visitor in
chat/list

Is it possible to perform this translation?
![Captura de Tela 2022-11-30 às 12 12 38](https://user-images.githubusercontent.com/9898902/204836026-d1170985-465b-4576-b003-8363c67be64a.png)
![Captura de Tela 2022-11-30 às 12 13 09](https://user-images.githubusercontent.com/9898902/204836060-69b227e0-3fc2-4c99-adea-5653296496f9.png)

**Comment by remdex** [comment]
This value is not translatable and indicates it's an anonymous visitor.

**Comment by GuilhermeDeLucca** [comment]
Yes, I understand that it indicates that you are an anonymous visitor!

I was hoping it could also be translated

Is there a possibility that it will be translated in the future?

Thanks for the answer!

**Reply by remdex** [reply]
No,
Because some extensions just check for this value to know was visitor known by the system or not.

**Reply by remdex** [reply]
If you website knows who is chatting with, you can just pass this value as you want :)

**Reply by GuilhermeDeLucca** [reply]
> No, Because some extensions just check for this value to know was visitor known by the system or not.

What a pity, it would be very user-friendly. 

(Visitatore) in Italian
(Visior) in Ingles

As in Ex.
![Captura de Tela 2022-11-30 às 13 01 30](https://user-images.githubusercontent.com/9898902/204847588-6e0c6abe-8674-4ab7-96b2-42692ccaed72.png)

**Comment by GuilhermeDeLucca** [comment]
Got it, imagine that the validations were done by <source> and not by <translation>    
    
    <message>
      <source>Visitor</source>
      <translation>Visitatore</translation>
    </message>

**Reply by remdex** [reply]
It won't be done. If you want it other than visitor you can just
1. Setup bot which would change on chart start to your string.
2. Pass your language nick. https://doc.livehelperchat.com/docs/custom-fields-and-prefill/

**Comment by GuilhermeDeLucca** [comment]
still unsuccessfu :/

![Captura de Tela 2022-11-30 às 22 10 18](https://user-images.githubusercontent.com/9898902/204942119-7cdefdcc-59c5-44cd-a554-b7cebf2936ab.png)
![Captura de Tela 2022-11-30 às 22 11 18](https://user-images.githubusercontent.com/9898902/204942124-a85c2c09-31ae-41e4-9915-faa2d55c3aab.png)

**Reply by remdex** [reply]
You need to update not message attribute, but chat.
